adapters/handlers/rest/clusterapi/indices.go

```diff
@@ -62,13 +66,18 @@ const (
 	cl = entschema.ClassNameRegexCore
 	sh = entschema.ShardNameRegexCore
 	ob = `[A-Za-z0-9_+-]+`
+	l  = "[0-9]{1,2}"
```

Comment:
In this case, what does `l` indicate? Does it have meaning beyond alphanumeric char?

Reply:
ahh got it. no change needed, just curious